### ISSUE TITLE
Error with empty Flag file on minisdm

### ISSUE THREAD

**Issue by KshitijAggarwal** [realfastvla/rfpipe#59]
Error when running search on a minisdm with `applyonlineflags = True` and an empty Flag.xml file. 

<img width="1039" alt="Screen Shot 2019-05-09 at 2 04 52 PM" src="https://user-images.githubusercontent.com/29838105/57483115-71516800-7263-11e9-856b-5304f5496e07.png">

**Comment by caseyjlaw** [comment]
I don't think this is related to an empty Flag.xml file, since I cannot reproduce it with another SDM like this.
Can you reproduce it and tell me which SDM you used?

**Comment by KshitijAggarwal** [comment]
Here is the SDM I used: 
`realfast_19A-393_TEST_B2319_000.58602.86878855324_1556572173889`

It is in the staging area as well.

**Comment by KshitijAggarwal** [comment]
I used the following prefs:

prefs={'gainfile': gainfile, 'saveplots': False, 'savenoise': False, 'savesols': False,
       'savecandcollection': True, 'savecanddata': True,
       'applyonlineflags': True, 'fftmode': 'cuda', 'clustercands': (4,3)}

st = rfpipe.state.State(sdmfile='/lustre/aoc/projects/fasttransients/staging/realfast_19A-393_TEST_B2319_000.58602.86878855324_1556572173889/', sdmscan=1, inprefs=prefs,
                        preffile=os.path.join(workdir, 'realfast.yml'))

**Comment by caseyjlaw** [comment]
I can't reproduce this in the development branch. Are you using that?

**Comment by KshitijAggarwal** [comment]
Yes, I am using the development branch, and am facing this issue with other SDM files as well.

**Comment by caseyjlaw** [comment]
Could you try updating lxml and installing the latest realfast code?

**Comment by KshitijAggarwal** [comment]
I updated lxml, and installed the latest development code, but I still get this error.

**Comment by caseyjlaw** [comment]
Oh, have you tried installing the latest `sdmpy`? It hasn't changed much, but the development branch in `realfastvla/sdmpy` repo may be more up to date.
Also, let me know if you were using the pip version of sdmpy. If that one is not working, then I should push a new version to pypi.
